src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -259,7 +258,7 @@ const styles = StyleSheet.create({
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also replaces a hardcoded color value with a constant, which is a good practice for the same reasons mentioned above.
```diff
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->